arch/xtensa/src/esp32s3/esp32s3_ble_adapter.c

```diff
@@ -896,7 +960,8 @@ static int IRAM_ATTR semphr_take_from_isr_wrapper(void *semphr, void *hptw)
 {
   *(int *)hptw = 0;
 
-  return esp_errno_trans(nxsem_trywait(semphr));
+  DEBUGPANIC();
```

Comment:
Why change to panic here? I think we can trywait from ISR

Reply:
Basically, because it isn't needed. We don't expect this function to be called by the BLE driver and we save IRAM space once we don't need to allocate the `nxsem_trywait` implementation there.

Reply:
Ok.